app/buck2/src/lib.rs

```diff
@@ -157,6 +157,7 @@ impl BeforeSubcommandOptions {
     }
 }
 
+#[rustfmt::skip]
```

Comment:
What is that for? We are using nightly rustfmt, maybe that is the issue?

Reply:
It moves the OSS deny comment around. I didn't know how robust that tooling was internally, so I went with skip instead. Check out the moved around one and let me know if it works. This was run with:
```
rustfmt 1.5.1-nightly (c8e6a9e 2023-01-23)
```

Reply:
Ah, I see it now: it moves `@oss-disable`.

Reply:
Can you please restore `rustfmt::skip`, and add a comment like "because it formats `@oss-disable` line differently in opensource and internal version?